cmd/minikube/cmd/start.go

```diff
@@ -1370,6 +1380,11 @@ func validateKubernetesVersion(old *config.ClusterConfig) {
 		exitIfNotForced(reason.KubernetesTooOld, "Kubernetes {{.version}} is not supported by this release of minikube", out.V{"version": nvs})
 	}
 
+	if _, ok := slowK8sVersions[nvs.String()]; ok {
```

Comment:
if i:= reason.MatchK8sIssues(); i != nil {
out.Box ( there is a known issue with X k8s verison : Issue :%i.IssueSummary, for more information see i.IssueURL)"
}

Reply:
done.